source/guide_openslides.rst

```diff
@@ -0,0 +1,221 @@
+.. highlight:: console
+
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+.. tag:: lang-python
+.. tag:: django
+.. tag:: presentations
+.. tag:: web
+.. tag:: assemblies
+.. tag:: meetings
+.. tag:: audience-business
+
+.. sidebar:: About
+
+  .. image:: _static/images/openslides.svg
+      :align: center
+
+##########
+OpenSlides
+##########
+
+.. tag_list::
+
+OpenSlides is a free, web-based presentation and assembly system for managing and projecting agenda, motions, and elections of assemblies.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>` and its package manager pip
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`Web Backends <web-backends>`
+
+License
+=======
+
+OpenSlides_ is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Install Python package
+----------------------
+Install the OpenSlides Python package:
+
+::
+
+ [isabell@stardust ~]$ pip3.9 install openslides --user
+ [...]
+ Running setup.py install for pyrsistent ... done
+ Running setup.py install for PyPDF2 ... done
+ Running setup.py install for roman ... done
+ Running setup.py install for websockets ... done
+ Running setup.py install for openslides ... done
+ [...]
+ [isabell@stardust ~]$
+
+Check if OpenSlides is installed by typing:
+
+::
+
+ [isabell@stardust ~]$ openslides --version
+ 3.3
+ [isabell@stardust ~]$
+
+Create configuration
+--------------------
+Run the following command to create the configuration:
+
+::
+
+ [isabell@stardust ~]$ openslides createsettings
+ Settings created at /home/isabell/.config/openslides/settings.py
+ [isabell@stardust ~]$ 
+
+Furthermore, populate the SQLite database by running the following command:
```

Comment:
Is it possible to use MySQL here? SQLite and slow is quite resource intensive in comparison, since it is lacking the cache in RAM.

Reply:
Using MySQL now.